db/Controllers/app.controllers.js

```diff
@@ -70,10 +71,17 @@ function addComment(req, res, next) {
   });
 }
 
+function getUsers(req, res, next) {
+  fetchUsers().then((users) => {
+    res.status(200).send({ users });
+  });
```

Comment:
make sure youre adding catch blocks to anywhere you are making an async request, just incase anything rejects :)